src/components/Products/ProductsFilter.js

```diff
@@ -0,0 +1,289 @@
+
+                </div>
+
+            </div>
+
+
+            <div className='mg-btm-lg'>
+
+                <Text classes='txt-md txt-cap mg-btm-xs'>size</Text>
+
+                <div className="grid grid-maxcols-3">
+
+                    <div className='flx flx-min-center'>
+
+                        <input type='checkbox' id='size-14' name="size-group" value='14' checked={isInputIncluded(14, filterState.sizeChecks)} onChange={(e) => handleSizeCheck(e)} className='mg-right-xs' />
+
+                        <label htmlFor='size-14' className="txt-cap">14</label>
+
+                    </div>
+
+                    <div className='flx flx-min-center'>
+
+                        <input type='checkbox' id='size-13' name="size-group" value='13' checked={isInputIncluded(13, filterState.sizeChecks)} onChange={(e) => handleSizeCheck(e)} className='mg-right-xs' />
+
+                        <label htmlFor='size-13' className="txt-cap">13</label>
+
+                    </div>
+
+                    <div className='flx flx-min-center'>
+
+                        <input type='checkbox' id='size-12' name="size-group" value='12' checked={isInputIncluded(12, filterState.sizeChecks)} onChange={(e) => handleSizeCheck(e)} className='mg-right-xs' />
+
+                        <label htmlFor='size-12' className="txt-cap">12</label>
+
+                    </div>
+
+                    <div className='flx flx-min-center'>
+
+                        <input type='checkbox' id='size-11' name="size-group" value='11' checked={isInputIncluded(11, filterState.sizeChecks)} onChange={(e) => handleSizeCheck(e)} className='mg-right-xs' />
+
+                        <label htmlFor='size-11' className="txt-cap">11</label>
+
+                    </div>
+
+                    <div className='flx flx-min-center'>
+
+                        <input type='checkbox' id='size-10' name="size-group" value='10' checked={isInputIncluded(10, filterState.sizeChecks)} onChange={(e) => handleSizeCheck(e)} className='mg-right-xs' />
+
+                        <label htmlFor='size-10' className="txt-cap">10</label>
+
+                    </div>
+
+                    <div className='flx flx-min-center'>
+
+                        <input type='checkbox' id='size-9' name="size-group" value='9' checked={isInputIncluded(9, filterState.sizeChecks)} onChange={(e) => handleSizeCheck(e)} className='mg-right-xs' />
+
+                        <label htmlFor='size-9' className="txt-cap">9</label>
+
+                    </div>
+
+                    <div className='flx flx-min-center'>
+
+                        <input type='checkbox' id='size-8' name="size-group" value='8' checked={isInputIncluded(8, filterState.sizeChecks)} onChange={(e) => handleSizeCheck(e)} className='mg-right-xs' />
+
+                        <label htmlFor='size-8' className="txt-cap">8</label>
+
+                    </div>
+
+                    <div className='flx flx-min-center'>
```

Comment:
Only the size number is changing. Try to extract this

Reply:
made the changes.